examples/ecommerce/tsconfig.json

```diff
@@ -10,6 +10,7 @@
     "esModuleInterop": true,
     "module": "esnext",
     "moduleResolution": "node",
+    "downlevelIteration": true,
```

Comment:
Not a big deal but I wonder if we can update the existing code instead of adding the rule?